src/main/java/com/aws/greengrass/clientdevices/auth/iot/Thing.java

```diff
@@ -147,4 +154,18 @@ public String getNamespace() {
     public Map<String, DeviceAttribute> getDeviceAttributes() {
         return Collections.singletonMap("ThingName", new WildcardSuffixAttribute(thingName));
     }
+
+    /**
+     * Updates the duration for which a certificate attachment can be trusted.
+     *
+     * @param newTrustDuration desired trust duration in hours
+     */
+    public static void updateMetadataTrustDurationHours(int newTrustDuration) {
+        metadataTrustDurationHours.set(newTrustDuration);
```

Comment:
Should we synchronize on this variable basically here and on the `isCertAttachementTrusted` method

Reply:
These updates to `AtomicInteger` are guaranteed to be thread-safe, without using explicit synchronization.